code/e2e-tests/framework-nextjs.spec.ts

```diff
@@ -21,6 +21,30 @@ test.describe('Next.js', () => {
     await new SbPage(page).waitUntilLoaded();
   });
 
+  test.describe('next/image', () => {
+    let sbPage: SbPage;
+
+    test.beforeEach(async ({ page }) => {
+      sbPage = new SbPage(page);
+    });
+
+    test('should lazy load images by default', async () => {
+      await sbPage.navigateToStory('frameworks/nextjs/Image', 'lazy');
+
+      const img = sbPage.previewRoot().locator('img');
+
+      expect(await img.evaluate<boolean, HTMLImageElement>((image) => image.complete)).toBeFalsy();
```

Comment:
A lazy loaded image element's complete property will be false if the image hasn't been loaded yet.